client/src/reducers/scoreSheet-fakedata.json

```diff
@@ -0,0 +1,660 @@
+{
+	"activities": [
+		{
+			"name": "Push Ups",
+			"value": "10"
+		}, 		
+		{
+			"name": "HS Push Ups",
+			"value": "50"
+		}, 		
+		{
+			"name": "Pull Ups",
+			"value": "50"
+		}, 		
+		{
+			"name": "Air Squats",
+			"value": "5"
+		}, 		
+		{
+			"name": "Distance Swam",
+			"value": "3500"
+		},
+		{
+			"name": "Distance Ran",
+			"value": "1000"
+		},		 
+		{
+			"name": "Distance Cycled",
+			"value": "400"
+		},
+		{
+			"name": "Distance Rowed",
+			"value": "400"
+		},		
+		{
+			"name": "1-Minute Planks",
+			"value": "200"
+		},
+		{
+			"name": "Low Intensity",
+			"value": "5000"
+		},		
+		{
+			"name": "High Intensity",
+			"value": "7000"
+		},
+		{
+			"name": "Sports",
+			"value": "3500"
+		},
+		{
+			"name": "Nutrition Bonus",
+			"value": "1000"
+		},
+		{
+			"name": "Stretch Bonus",
+			"value": "1000"
+		},
+		{
+			"name": "Tabata Bonus",
+			"value": "1500"
+		}		
+	],
+	"users": [
+		{
+			"username": "Darren",
+			"id": 1,
+			"total": "80000",
+			"head2head": "30",
+			"activities": [
+				{	"name": "Push Ups",
+					"value": "20"
+				},
+				{	"name": "HS Push Ups",
```

Comment:
In case we have different leagues with the same named exercise, do you think we should someday create a key for these exercises? What if "running" allows for commute-walking in one league and not in another league? Would we want folks to be able to name their exercises "running" but differentiate them with a key? The key could connect exercises across leagues, but if someone wanted to redefine "Running"'s definition, their new exercise would get a new key. 

Maybe we would have a "Running" exercise that's standard. They could create a custom "Running" exercise, but when they are creating a new league they can only use the "Running" exercise we've defined or their custom exercise. This prevents them from being inundated with 7 kinds of "Running" exercises that other folks have created, but it also allows us to have a running exercise we can someday treat as a KPI that will compare apples-to-apples across leagues.

Reply:
Let me know if you'd like me to create an issue or discuss at our meeting. If you'd like to create the issue, that's cool also.